service/collector/app_resource.go

```diff
@@ -43,14 +61,40 @@ func NewAppResource(config AppResourceConfig) (*AppResource, error) {
 	return c, nil
 }
 
+func (c *AppResource) collectAppStatus(ctx context.Context, ch chan<- prometheus.Metric) {
+	r, err := c.g8sClient.ApplicationV1alpha1().Apps("").List(metav1.ListOptions{})
+	if err != nil {
+		c.logger.LogCtx(ctx, "level", "error", "message", "could not get apps", "stack", fmt.Sprintf("%#v", err))
```

Comment:
If we error here, is `r` `nil`? Because then you'll panic later

Reply:
I fixed it, thanks for notice.